### ISSUE TITLE
unplugin-vue-macros 2.5.0 bug

### ISSUE THREAD

**Issue by LYlanfeng** [vue-macros/vue-macros#502]
componet：
```javascript
<el-form
   inline // boolean attr, will add el-form--inline class name in componet
>
</el-form>
```
unplugin-vue-macros 2.4.9 compile result:
```javascript
<form class="el-form el-form--default el-form--label-right el-form--inline ly-e-form">
// el-form--inline === inline
```
unplugin-vue-macros 2.5.0 compile result:
```javascript
<form class="el-form el-form--default el-form--label-right ly-e-form" checked="true">
// checked="true" === inline, miss class attr  el-form--inline because componet's attr inline is invalid
```

**Comment by sxzz** [comment]
We temporarily close this due to the lack of enough information.
Please provide a [minimal reproduction](https://stackoverflow.com/help/minimal-reproducible-example) to reopen the issue.
Thanks.

> [Why reproduction is required](https://antfu.me/posts/why-reproductions-are-required)

**Comment by sxzz** [comment]
I confirmed that it's a bug.

P.S The previous comment is submitted by automatic script.

**Comment by sxzz** [comment]
Fixed in `unplugin-vue-macros@2.5.1`. Thanks for your feedback.
